sdk/test-utils/recorder/src/baseRecorder.ts

```diff
@@ -0,0 +1,344 @@
+import fs from "fs-extra";
+import nise from "nise";
+import queryString from "query-string";
+import { isBrowser, blobToString } from "./utils";
+import { customConsoleLog } from "./customConsoleLog";
+
+const env = isBrowser() ? (window as any).__env__ : process.env;
+const isRecording = env.TEST_MODE === "record";
+const isPlayingBack = env.TEST_MODE === "playback";
+
+let nock: any;
+if (!isBrowser() && (isRecording || isPlayingBack)) {
+  nock = require("nock");
+}
+
+if (isBrowser() && isRecording) {
+  customConsoleLog();
+}
+
+if (isPlayingBack) {
+  // Providing dummy values to avoid the error
+  env.ACCOUNT_NAME = "fakestorageaccount";
+  env.ACCOUNT_KEY = "aaaaa";
+  env.ACCOUNT_SAS = "aaaaa";
+  env.STORAGE_CONNECTION_STRING = `DefaultEndpointsProtocol=https;AccountName=${
+    env.ACCOUNT_NAME
+  };AccountKey=${env.ACCOUNT_KEY};EndpointSuffix=core.windows.net`;
+}
+
+const skip = [
+  // Abort
+  "browsers/aborter/recording_should_abort_after_aborter_timeout.json"
+];
```

Comment:
Eventual plan is to remove the skip list - https://github.com/azure/azure-sdk-for-js/issues/4084

Reply:
https://github.com/Azure/azure-sdk-for-js/pull/4336